crates/autopilot/src/infra/persistence/mod.rs

```diff
@@ -484,6 +486,72 @@ impl Persistence {
                 )
                 .await?;
             }
+
+            database::jit_orders::upsert_orders(
+                &mut ex,
+                &jit_orders
+                    .into_iter()
+                    .map(|jit_order| database::jit_orders::JitOrder {
+                        block_number,
+                        log_index,
+                        uid: ByteArray(jit_order.uid.0),
+                        owner: ByteArray(jit_order.uid.owner().0 .0),
+                        creation_timestamp: chrono::DateTime::from_timestamp(
+                            jit_order.created as i64,
```

Comment:
As @m-lord-renkse rightly noted in another PR, let's use `from()` function in case `created` field type is changed and the compiler will be aware of it.
```suggestion
                            i64::from(jit_order.created),
```

Reply:
Is this only preferred for i64, or we also avoid any `map(Into::into)` cases?

Reply:
resolved